production/Makefile

```diff
@@ -16,7 +16,9 @@ CLOUD_INITS := \
 	cloud-init/travis-worker-gce-org-prod2 \
 	cloud-init/travis-worker-gce-org-prod3 \
 	cloud-init/travis-worker-gce-com-staging \
-	cloud-init/travis-worker-gce-org-staging
+	cloud-init/travis-worker-gce-org-staging \
+	cloud-init/travis-tmate-edge-org-staging \
```

Comment:
These probably shouldn't have "org" in the name, since they are shared between .com and .org.